packages/react-components/src/components/MIMap/MIMap.jsx

```diff
@@ -0,0 +1,66 @@
+import { useEffect, useState } from 'react';
+import PropTypes from 'prop-types';
+import MapboxMap from './MapboxMap/MapboxMap';
+import GoogleMapsMap from './GoogleMapsMap/GoogleMapsMap';
+
+const mapTypes = {
+    Google: 'google',
+    Mapbox: 'mapbox'
+};
```

Comment:
Would it make sense to move this outside as a constant? We might need to use it along the way, at least as far as I remember, we did that in the previous implementation.

Reply:
I'll do it when we need it in more places 😄